### ISSUE TITLE
Duplicate content and a missing word on "Help users to Navigate a service"

### ISSUE THREAD

**Issue by JodiB-TPR** [alphagov/govuk-design-system#4089]
## What
Under the second WCAG 2.2 tag in the content, you have duplicated content. Is it meant to be there or was it meant to say something else?

"...and **must always link to the same place**.

**For example, a header link to “Get help with this service” must go to the same place as similar header links elsewhere in your service.** This is to comply with WCAG 2.2 success criterion [3.2.6 Consistent help](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html).

Help links **must always link to the same place. For example, a header link to “Get help with this service” must go to the same place as similar header links elsewhere in your service.**

![image](https://github.com/user-attachments/assets/3e10a8d4-c9db-4f9b-bfbf-cf6a45d3c15a)

There also appears to be a word missing ("the") in the first sentence of the second paragraph under the heading "Adding other header and navigation elements". It should probably read "...should start with **the** most..."

![image](https://github.com/user-attachments/assets/bde8c58a-69f9-4642-b51b-26aa79faad25)

**Comment by calvin-lau-sig7** [comment]
Thanks for the proofread Jodi, looking to shake these down today.

**Comment by calvin-lau-sig7** [comment]
Resolved as part of work in https://github.com/alphagov/govuk-design-system/pull/4092, happy days and thanks again!
